packages/kbn-journeys/journey/journey_ftr_harness.ts

```diff
@@ -333,33 +332,33 @@ export class JourneyFtrHarness {
   private telemetryTrackerCount = 0;
 
   private trackTelemetryRequests(page: Page) {
-    const id = ++this.telemetryTrackerCount;
-
-    const requestFailure$ = Rx.fromEvent<Request>(page, 'requestfailed');
     const requestSuccess$ = Rx.fromEvent<Request>(page, 'requestfinished');
     const request$ = Rx.fromEvent<Request>(page, 'request').pipe(
       Rx.takeUntil(
         this.pageTeardown$.pipe(
           Rx.first((p) => p === page),
           Rx.delay(3000)
-          // If EBT client buffers:
-          // Rx.mergeMap(async () => {
-          //  await page.waitForFunction(() => {
-          //    // return window.kibana_ebt_client.buffer_size == 0
-          //  });
-          // })
         )
       ),
-      Rx.mergeMap((request) => {
+      Rx.mergeMap((request: Request) => {
         if (!request.url().includes('telemetry-staging.elastic.co')) {
           return Rx.EMPTY;
         }
 
-        this.log.debug(`Waiting for telemetry request #${id} to complete`);
-        return Rx.merge(requestFailure$, requestSuccess$).pipe(
-          Rx.first((r) => r === request),
+        const id = ++this.telemetryTrackerCount;
+        this.log.info(`Waiting for telemetry request #${id} to complete`);
+        return Rx.of(requestSuccess$).pipe(
+          Rx.timeout(60_000),
+          Rx.catchError((error) => {
+            if (error instanceof Error && error.name === 'TimeoutError') {
+              this.log.error(`Timeout error occurred: ${error.message}`);
+            }
+            // Rethrow the error if it's not a TimeoutError
+            return Rx.throwError(() => new Error(error));
+          }),
           Rx.tap({
-            complete: () => this.log.debug(`Telemetry request #${id} complete`),
+            complete: () => this.log.info(`Telemetry request #${id} complete`),
+            error: (err) => this.log.error(`Telemetry request was not processed: ${err.message}`),
```

Comment:
I made a few changes to make it work correctly:
- `telemetryTrackerCount` was always returning `1` since it was misplaced before observables.
- delete `requestFailure$` since we don't have to do anything in case of Telemetry request failure event. We fully rely on kibana service to things the right way.
- adding timeout of 60 sec to issue error notification in case of no new emitted events;
- log messages on both complete & error

```
 └-> Go to Transform Wizard
     └-> "before each" hook: global before each for "Go to Transform Wizard"
     │ debg APM DEBUG start span {"span":"fe0fad39231fa33e","parent":"9cf0f1c9d8df9a6a","trace":"ada4a00b5cac22401cd1ed060e28e4f2","name":"step: Go to Transform Wizard","type":"step","subtype":null,"action":null}
     │ debg APM DEBUG supersedeWithSpanRunContext(<Span fe0fad39231fa33e>)
     │ info Waiting for telemetry request #2 to complete
     │ info Telemetry request #2 complete
     │ info Waiting for telemetry request #3 to complete
     │ info Telemetry request #3 complete
     │ info Waiting for telemetry request #4 to complete
     │ info Telemetry request #4 complete
     │ debg APM DEBUG ended span {"span":"fe0fad39231fa33e","parent":"9cf0f1c9d8df9a6a","trace":"ada4a00b5cac22401cd1ed060e28e4f2","name":"step: Go to Transform Wizard","type":"step","subtype":null,"action":null}
     │ debg APM DEBUG addEndedSpan(step: Go to Transform Wizard)
     │ debg APM DEBUG encoding span {"span":"fe0fad39231fa33e","parent":"9cf0f1c9d8df9a6a","trace":"ada4a00b5cac22401cd1ed060e28e4f2","name":"step: Go to Transform Wizard","type":"step"}
     └- ✓ pass  (1.0m)
```